mule-packager/src/test/java/org/mule/tools/api/validation/AbstractProjectValidatorTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.api.validation;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.*;
+
+import org.junit.rules.ExpectedException;
+import org.mule.tools.api.exception.ValidationException;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mule.tools.api.util.Project;
+
+public class AbstractProjectValidatorTest {
```

Comment:
Please rename
AbstractProjectValidatorTest  -> MuleProjectValidator

Reply:
O now I see what you did here, you can either move this tests to the test of one of the implementations and remove this class or maybe instead of using one of the implementations create an inner class StubProjectValidator extends ProjectValidator 
So there are no confusions what's under testing

Reply:
ok! I guess the second option with mock or spy 👍